config.gradle

```diff
@@ -8,7 +8,7 @@ nativeUtils.withCrossLinuxArm64()
 nativeUtils {
     wpi {
         configureDependencies {
-            wpiVersion = "2025.+"
+            wpiVersion = wpilibVersion
             opencvYear = "frc2024"
```

Comment:
Can you also add the same things for opencvYear and opencvVersion? That way the versions that are shared are in a common place.

Reply:
Yes, but that was out of scope of this PR